test/thermo/PengRobinsonMFTP_Test.cpp

```diff
@@ -0,0 +1,210 @@
+        standardConcs[0] = test_phase->standardConcentration(0);
+        standardConcs[2] = test_phase->standardConcentration(2);
+        test_phase->getActivityConcentrations(&activityConcentrations[0]);
+
+        EXPECT_NEAR(standardConcs[0] * r * activityCoeffs[0], activityConcentrations[0], 1.e-6);
+        EXPECT_NEAR(standardConcs[2] * (1-r) * activityCoeffs[2], activityConcentrations[2], 1.e-6);
+    }
+}
+
+TEST_F(PengRobinsonMFTP_Test, setTP)
+{
+    // Check to make sure that the phase diagram is accurately reproduced for a few select isobars
+
+    // All sub-cooled liquid:
+    const double p1[6] = {
+        1.666169531425807e+002,
+        1.610359415729891e+002,
+        1.561558755014150e+002,
+        1.518163520394436e+002,
+        1.479077426738997e+002,
+        1.443514857504623e+002        
+    };
+    // Phase change between temperatures 4 & 5:
+    const double p2[6] = {
+        7.085417135252475e+002,
+        6.715912546208409e+002,
+        6.252218774469104e+002,
+        5.555249258326570e+002,
+        3.218596258615833e+002,
+        2.791272952614171e+002
+    };
+    // Supercritical; no discontinuity in rho values:
+    const double p3[6] = {
+        7.695965571714201e+002,
+        7.455746209641901e+002,
+        7.196124816622006e+002,
+        6.912592525839008e+002,
+        6.598814961519255e+002,
+        6.245690038776825e+002
+    };
+
+    for(int i=0; i<6; ++i)
+    {
+        const double temp = 294 + i*2;
+        set_r(0.99);
+        test_phase->setState_TP(temp, 5542027.5);
+        EXPECT_NEAR(test_phase->density(),p1[i],1.e-8);
+
+        test_phase->setState_TP(temp, 7389370.);
+        EXPECT_NEAR(test_phase->density(),p2[i],1.e-8);
+
+        test_phase->setState_TP(temp, 9236712.5);
+        EXPECT_NEAR(test_phase->density(),p3[i],1.e-8);
+    }
+}
+
+TEST_F(PengRobinsonMFTP_Test, getPressure)
+{
+    // Check to make sure that the P-R equation is accurately reproduced for a few selected values
+
+    /* This test uses CO2 as the only species.
+    *  Values of a_coeff, b_coeff are calculated based on the the critical temperature and pressure values of CO2 as follows:
+    *       a_coeff = 0.457235(RT_crit)^2/p_crit
+    *       b_coeff = 0.077796(RT_crit)/p_crit
+    *  The temperature dependent parameter in P-R EoS is calculated as
+    *       \alpha = [1 + \kappa(1 - sqrt{T/T_crit}]^2
+    *  kappa is a function calulated based on the accentric factor.
+    */
+
+    double a_coeff = 3.958134E+5;
+    double b_coeff = 26.6275/1000;
+    double acc_factor = 0.228;
+    double pres_theoretical, kappa, alpha, mv;
+    const double rho = 1.0737; 
+    const double Tcrit = test_phase->critTemperature();
+
+    //Calculate kappa value
+    if (acc_factor <= 0.491) {
+        kappa = 0.37464 + 1.54226*acc_factor - 0.26992*acc_factor*acc_factor;
+    }
+    else {
+        kappa = 0.379642 + 1.487503*acc_factor - 0.164423*acc_factor*acc_factor + 0.016666*pow(acc_factor,3);
+    }
```

Comment:
Given that `acc_factor` is a constant, this second branch can never be executed.

Reply:
I agree.  I have removed the redundant code.